docs/resources/vault_secrets_integration_azure.md

```diff
@@ -0,0 +1,75 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "hcp_vault_secrets_integration_azure Resource - terraform-provider-hcp"
+subcategory: ""
+description: |-
+  The Vault Secrets Azure integration resource manages an Azure integration.
```

Comment:
This feel incomplete.

Reply:
Similar to the above comment, this is a standard message across integrations.